cylc/flow/task_events_mgr.py

```diff
@@ -768,7 +768,6 @@ def _process_message_submit_failed(self, itask, event_time):
             "submit_status": 1,
         })
         job_d = get_task_job_id(itask.point, itask.tdef.name, itask.submit_num)
-        self.job_pool.set_job_attr(job_d, 'batch_sys_job_id', None)
```

Comment:
@kinow - There was no point in having this anyway! (I think) Because if something submit-fails then it has no `batch_sys_job_id` and if it does have one, we don't want to wipe that information out.

Reply:
Makes sense. Less one warning in the logs then! Thanks @dwsutherland !